configuration/blockly/rules-blockly-run-and-process.md

```diff
@@ -95,6 +96,9 @@ Here is a list of possible values that can be retrieved.
 - triggered channel - String
 - triggered event - String
 
+Note on "event available": All except _rule UID_ expect an event context to be available.
```

Comment:
```suggestion
Note on "event available": All values except _rule UID_ expect an event context to be available.
```

Since openHAB 4.1 the event context should always be available, even on manual execution, see https://github.com/openhab/openhab-core/pull/2965.
Unfortunately I did not remember that when merging that UI PR, we should probably think about how to get rid of that value.